auth_from_http_remote_user/models/res_users.py

```diff
@@ -0,0 +1,76 @@
+##############################################################################
+#
+#    Author: Laurent Mignon
+#    Copyright 2014 'ACSONE SA/NV'
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as
+#    published by the Free Software Foundation, either version 3 of the
+#    License, or (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+
+import odoo
+from odoo import fields, models
+from odoo.addons.auth_from_http_remote_user import utils
+
+
+class Users(models.Model):
+    _inherit = 'res.users'
+
+    sso_key = fields.Char('SSO Key', size=utils.KEY_LENGTH,
+                          readonly=True)
+
+    def copy(self, default=None):
+        default = default = {} if default is None else default
+        default['sso_key'] = False
+        return super(Users, self).copy(default)
```

Comment:
The override of `copy` should be replaced by the `copy=False` argument in the definition of the `sso_key` field.

Reply:
Changed